client/src/components/AddTableModal/AddTableModal.js

```diff
@@ -140,7 +140,8 @@ const AddTableModal = _ => {
                   disabled={isLoading}
                   onClick={event => {
                     handleSubmitTable(event)
-                    window.location.reload(false)
+                    handleClose()
```

Comment:
after submission is complete, close the modal.
Removed the page refresh function.